app/invocation/workflow_rerun_button.tsx

```diff
@@ -70,7 +70,20 @@ export default class WorkflowRerunButton extends React.Component<WorkflowRerunBu
           visibility: this.props.model.buildMetadataMap.get("VISIBILITY") || "",
         })
       )
-      .then((response) => router.navigateTo(`/invocation/${response.invocationId}`))
+      .then((response) => {
+        let invocationId = "";
+        response.actionStatuses.forEach(function (actionStatus, _) {
+          if (actionStatus.actionName == configuredEvent.actionName) {
+            invocationId = actionStatus.invocationId;
+            return
+          }
+        });
+        if (invocationId != "") {
```

Comment:
It's good to get in the habit of using `===` and `!==` for equality / inequality checking in JS/TS, since `==` and `!=` do type coercion that can have unintended effects.